### PR TITLE
Reorganizing the Operation > Monitoring pages

### PR DESCRIPTION
**Pull request by tcnghia**
## Description

Reorganizing the Operation > Monitoring pages to follow the structure suggested in https://github.com/dapr/docs/issues/1182

## Issue reference

Fixes https://github.com/dapr/docs/issues/1182

### CODE REVIEW

**Comment by tcnghia** [comment]
@orizohar can you please help review & merge ? thanks

File: daprdocs/content/en/operations/monitoring/tracing/open-telemetry-collector.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ type: docs
 title: "Using OpenTelemetry Collector to collect traces"
 linkTitle: "OpenTelemetry"
 weight: 1000
-description: "How to use Dapr to push trace events to Azure Application Insights, through the OpenTelemetry Collector."
+description: "How to use Dapr to push trace events through the OpenTelemetry Collector."
```
Comment (orizohar) [inline daprdocs/content/en/operations/monitoring/tracing/open-telemetry-collector.md]: Currently this is still App Insights specific. We need to split this into 2 files. That can happen in a separate PR but if so, lets keep the description as is.

File: daprdocs/content/en/operations/monitoring/logging/_index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Logging"
+linkTitle: "Logging"
+weight: 400
+description: "How to setup loggings for Dapr sidecar, and your application"
+---
```
Comment (orizohar) [inline daprdocs/content/en/operations/monitoring/logging/_index.md:7]: We should add interlinking both to and from the "Troubleshooting --> Logs" document

File: daprdocs/content/en/operations/monitoring/logging/_index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+type: docs
+title: "Logging"
+linkTitle: "Logging"
+weight: 400
```
Comment (orizohar) [inline daprdocs/content/en/operations/monitoring/logging/_index.md:5]: This page should have weight so "Metrics" is higher

**Review by orizohar — CHANGES_REQUESTED** [review]
As part of the re-org we need to remove or move some of the pages under "developing-applications/building-blocks/observability/" right now it seems like we have duplicates.

File: daprdocs/content/en/operations/monitoring/tracing/open-telemetry-collector.md (hunk 1 above)
Comment (tcnghia) [reply daprdocs/content/en/operations/monitoring/tracing/open-telemetry-collector.md]: I am separating the PRs. So reverted the description change.

File: daprdocs/content/en/operations/monitoring/logging/_index.md (hunk 2 above)
Comment (tcnghia) [reply daprdocs/content/en/operations/monitoring/logging/_index.md:7]: Done

File: daprdocs/content/en/operations/monitoring/logging/_index.md (hunk 3 above)
Comment (tcnghia) [reply daprdocs/content/en/operations/monitoring/logging/_index.md:5]: Done

**Review by orizohar — APPROVED** [review]

**Pull request merged**